packages/@react-spectrum/searchwithin/docs/SearchWithin.mdx

```diff
@@ -0,0 +1,66 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/searchwithin';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/searchwithin/package.json';
+
+```jsx import
+import {SearchWithin} from '@react-spectrum/searchwithin';
+import {SearchField} from '@react-spectrum/searchfield';
+import {Picker, Item} from '@react-spectrum/picker';
+```
+
+---
+category: Forms
+keywords: [search within]
+after_version: 3.0.0-alpha.0
```

Comment:
Since the version in package.json is already greater than this, it would go out immediately upon merging this PR.

```suggestion
after_version: 3.0.0
```

Reply:
Great catch, thanks